docs/team/charleslee01.md

```diff
@@ -11,28 +11,41 @@ command box provided.
-  * The EditOwnerCommand and EditDogCommand class are implemented such that it extends from the EditEntityCommand.
-  * In the process, I had to build `EditEntityCommandParser`, `EditOwnerCommandParser` and `EditDogCommandParser`to allow the proper functioning of this command.
+* **Code Refactoring**: Code Quality Improvements
+  * Apply assertions to ensure maintain defensive coding.
+  * Update Javadoc comments so that it fits the current context.
+  * Improve code quality during development.
+  * Ensure the MESSAGE_USAGE is synchronised for all command.
+  
+* **Code Testing**: Update and improve test cases
+  * Contributed significantly to testing.
+  * Wrote test cases related to edit command, edit command parser, descriptor test for each entities and dog class.
+  * Added descriptor builders such as `EditEntityDescriptorBuilder` etc., which allows the testing of edit commands.
+  * Update the comments of old testcases.
 
 * **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s2.github.io/tp-dashboard/?search=CharlesLee01&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2021-02-19)
 
 * **Project management**:
+  * Note down project discussion on collaboration document.
 
-* **Enhancements to existing features**:
+* **Enhancements to existing features**: Edit Command and GUI Help Window
   * Allow edit command to work for three different entities: Dog, Owner and Program.
-
+  * Implemented the EditEntityCommand, EditOwnerCommand and EditDogCommand that edits owners and dogs in the database.
+  * The EditOwnerCommand and EditDogCommand class are implemented such that it extends from the EditEntityCommand.
+  * In the process, I had to build `EditEntityCommandParser`, `EditOwnerCommandParser` and `EditDogCommandParser`to allow the proper functioning of this command.
+  * Ensure help window contains all possible commands for Pawbook.
+  
 * **Documentation**:
   * User Guide:
-    * Contributed significant portions in initial layout of the user guide. Navigating the User Guide,
-      Edit Command and Command Summary.
-    * Maintain the consistency of the user guide. Make sure each command format is accurate.
-    * Use icons such as :bulb: and :heavy_exclamation_mark: to notify readers about possible warnings and hints when using Pawbook.
+    * Contributed significant portions in initial layout of the user guide. Navigating the User Guide, Edit Command and Command Summary.
+    * Maintain the consistency of the user guide. For instance, ensure the command formats are accurate.
+    * Ensure that sufficient examples and explanations are given so that the exact features of each command are clear to users. 
+    * Use icons such as :bulb: to notify readers about possible warnings and :heavy_exclamation_mark: to show the hints when using Pawbook.
```

Comment:
the bulb is for hints and the heavy exclamation mark is for warnings

Reply:
Yea, accidentally switched it during rephrasing. Thx!